cmd/tap-fitter/main.go

```diff
@@ -4,8 +4,11 @@ import (
 	"fmt"
 	"net/http"
 	"os"
+	
+	"github.com/release-engineering/tap-fitter/generation"
 
 	"github.com/operator-framework/operator-registry/alpha/template/composite"
+	"github.com/operator-framework/operator-registry/cmd/opm/internal/util"
```

Comment:
This doesn't seem to be used and can't be imported as it's an internal package.

Reply:
yup leftover artifact from when i was trying to see if it was worth making a change to export from the fbc tooling - removing now